db/migrate/20150128185746_seed_new_storytime_actions_and_permissions.rb

```diff
@@ -0,0 +1,9 @@
+class SeedNewStorytimeActionsAndPermissions < ActiveRecord::Migration
+  def up
+    Storytime::Action.seed
+    Storytime::Permission.seed
+  end
```

Comment:
Why do we need this migration? Were no actions/permissions added for the notifications stuff? If so, we should add these 2 lines to the add_notification_fields_to_storytime_post migration and delete this one.